deployments/deploy-ssh.sh

```diff
@@ -149,6 +153,22 @@ if [ $timeout -gt 600 ]; then
   describe_pod_then_exit
 fi
 
+# List of additional kubeconfigs of k8s clusters to register with px-backup, px-dr
+FROM_FILE=""
```

Comment:
Can u check things which are already in master branch are showing up here?

Reply:
Sure

Reply:
I've just put it above in them PR.